crates/bevy_render/src/color/mod.rs

```diff
@@ -683,6 +683,17 @@ impl Color {
         }
     }
 
+    /// Converts a `Color` to a `[u8; 4]` from sRGB colorspace
+    pub fn as_rgba_u8(&self) -> [u8; 4] {
+        let [r, g, b, a] = self.as_rgba_f32();
+        [
+            (r * 255.0) as u8,
```

Comment:
For those wondering like me, values less than `0` will cast to `0` and values greater than `255` will cast to `255`. So this should be okay since we expect each component to be in the range `[0, 1]` anyways.

Reply:
> `0` will cast to `0` and values greater than `255` will cast to `255`. 

I didn't actually know that but that's good to know haha